From 111e5d0311f174592ccecee2af11067103abaee7 Mon Sep 17 00:00:00 2001
From: aleks <aleks.stier@icloud.com>
Date: Sun, 11 Aug 2019 03:15:26 +0200
Subject: [PATCH] Make right-click paste

Pressing right-click pastes from the primary-selection.
If combined with the clipboard-patch right-click pastes from the clipboard.
Middle-click does nothing.
---
 x.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
--- a/x.c.orig	2020-08-23 13:47:47.168674708 +0300
+++ b/x.c	2020-08-23 13:51:48.550559910 +0300
@@ -698,7 +698,9 @@
 
 	if (mouseaction(e, 1))
 		return;
-	if (e->xbutton.button == Button1)
+    if (e->xbutton.button == Button3)
+        selpaste(NULL);
+    else if (e->xbutton.button == Button1)
 		mousesel(e, 1);
 }
 
-- 
2.22.0

